front-end/src/app/app.component.ts

```diff
@@ -1,10 +1,43 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
+class Task {
+  title = '';
+  comment = '';
+
+  constructor(titl: string, comm: string) {
+    this.title = titl;
+    this.comment = comm;
+  }
+}
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
```

Comment:
Az app komponensbe nem szoktunk ilyen jellegű logikát rakni, de most maradhat így. Később lesz érdekes, mikor már nem 1 felületből/viewból álló webappot fejlesztesz. Ilyen esetekben a routert kell használni, ami leegyszerűsítve egy adott url-en egy adott komponenst fog megjeleníteni.
ha érdekel ezekről itt találsz többet:
- https://angular.io/guide/routing-overview
- https://angular.io/tutorial/tour-of-heroes/toh-pt5